Services/DataBaseService/Repositories/UserCredentialRepository.cs

```diff
@@ -32,6 +33,20 @@ public Guid Create(UserEmailPassword data)
                 throw new Exception(e.Message + "\nCan`t add user");
             }
         }
+
+        public void Delete(Guid id)
```

Comment:
Where is id null check?

Reply:
It should check UserService

Reply:
Yeah. But we talked about this. What if user service will be rewrite? Why do we have to hope on other services? Also, I don't see something terrible in one more null check

Reply:
It is the task User Service and we trust UserService